tests/test_local_settings.py

```diff
@@ -149,3 +159,14 @@ def test_pod_mutation_hook(self):
             settings.pod_mutation_hook(pod)
 
             assert pod.namespace == 'airflow-tests'
+
+    def test_custom_policy(self):
```

Comment:
This code doesn't test any DagBag logic, just the settings code (which is already tested) and the code of the SETTINGS_FILE_CUSTOM_POLICY, but it is part of tests.  So there is no test that verifies the actual changes you have made.

I would be happy if you would add one test based on ``test_rules.py``.